script/Ciel/Day 9/09_01_CIEL09_1_1.txt

```diff
@@ -68,7 +68,7 @@ As I do―――
 -- Page 76, Offset 22329.
 -- たしかに後はつかえてるしぃ？　楽しい日々はもうおしまい、ぬっるい休憩はいい加減ここまでかもねぇ？」
 // ぬっるい休憩 is a strange combination.
-You said you had 'places to be', right?
+You said there's more people in line, right?
```

Comment:
maybe something like this would be heavy-handed enough
> Listen, there's somebody after me. We can catch up all you want next time.
You said there's %{i}someone coming after you%{/i}, right?